.github/pull_request_template.md

```diff
@@ -0,0 +1,36 @@
+## Proposed changes
+
+Include description and references to tickets or issues that this PR is related to
+
+## Checklist prior to adding reviewers
+
+_Put an `x` in the boxes that apply._
+
+- [ ] I have run this successfully locally using `./build.ps1 compose`
+- [ ] I have added tests for my code and/or updated existing tests where my changes impacts them, or there is a valid explanation for not including tests in this PR
+- [ ] I have added necessary documentation (if appropriate)
+- [ ] I created this branch from main and have integrated the latest changes
+- [ ] I conformed to the [Angular Commit Message Format](https://gist.github.com/brianclements/841ea7bffdb01346392c#commit-message-format) for all commit messages in this PR
+- [ ] I have ensured that the changes are backwards compatible, or I have ensured that `BREAKING CHANGE: <comment>` is included in the body of a commit message where backwards compatibility is broken
```

Comment:
There must be absolutely no breaking changes in the commit. Any such changes so would require extensive comms.

Reply:
At some point a service is going to need to be able to handle breaking changes. The typical process would be to mark a public interface as deprecated as part of a current major release, and then as part of the next major release, to then remove support for it once either confirmation from all clients have catered for the breaking change or the time provided to clients as part of the deprecation notice has elapsed.